pyproject.toml

```diff
@@ -57,7 +57,7 @@ tests = [
   "pytest_voluptuous",
   "dpath",
   "transformers[torch]",
-  "keras<3"
+  "tf-keras"
```

Comment:
See https://github.com/iterative/dvclive/pull/832#discussion_r1672621506. This should work in huggingface while still allowing us to run keras tests with keras 3.